webtau-groovy/src/main/groovy/com/twosigma/webtau/cli/WebTauCliApp.groovy

```diff
@@ -114,7 +112,10 @@ class WebTauCliApp implements StandaloneTestListener, ReportGenerator {
     }
 
     private List<Path> testFiles() {
-        return cliConfigHandler.testFiles.collect { Paths.get(it) }
+        return cliConfigHandler.testFiles.collect { fileName ->
+            def path = Paths.get(fileName)
+            return path.isAbsolute() ? path : cfg.workingDir.resolve(path)
```

Comment:
Thinking out loud, not convinced it's a good idea but... is it worth having a helper function on `WebTauConfig` to do this absolute/relative thing?  It's not the only place you do the exact same tertiary check.

Reply:
Was thinking the same. Makes sense to me.